feature-tests/utils/ais-events-array.ts

```diff
@@ -0,0 +1,147 @@
+export const aisEvents = {
+  suspendNoAction: [
```

Comment:
is there a reason why each of the elements in this object is an array containing one element? I think the array wrapping can be removed..

Reply:
Thanks Chiara, will resolve this in a future PR https://govukverify.atlassian.net/browse/ATB-1208